doc/source/docker/run-quickstart-examples-docker-compose.rst

```diff
@@ -0,0 +1,175 @@
+   run. The path should be relative to the location of the Docker Compose files:
+
+   .. code-block:: bash
+
+      $ export PROJECT_DIR=../../../examples/quickstart-sklearn-tabular
+
+#. Build and start the services using the following command:
+
+   .. code-block:: bash
+
+      $ docker compose -f compose.yml up --build -d
+
+#. Follow the logs of the SuperExec service:
+
+   .. code-block:: bash
+
+      $ docker compose logs superexec -f
+
+Step 2: Run the Quickstart Example (Terminal 2)
+-----------------------------------------------
+
+#. Change the directory to the quickstart example:
+
+   .. code-block:: bash
+
+      $ cd flower/examples/quickstart-sklearn-tabular
+
+#. Append the following lines to the end of the ``pyproject.toml`` file and save it:
+
+   .. code-block:: toml
+      :caption: pyproject.toml
+
+      [tool.flwr.federations.docker-compose]
+      address  =  "127.0.0.1:9093"
+      insecure  = true
+
+#. Run the example:
+
+   .. code-block:: bash
+
+      $ flwr run . docker-compose
+
+That is all it takes! You can monitor the progress of the run in the first terminal.
+
+Run a Different Quickstart Example
+----------------------------------
+
+#. To run a different quickstart example, such as ``quickstart-pytorch``, switch back to the first
+   terminal and update the ``PROJECT_DIR`` environment variable:
+
+   .. code-block:: bash
+      :caption: Terminal 1
+
+      $ export PROJECT_DIR=../../../examples/quickstart-pytorch
+
+#. Rebuild and restart the services and follow the SuperExec logs:
+
+   .. code-block:: bash
+      :caption: Terminal 1
+
+      $ docker compose -f compose.yml up --build -d --force-recreate
+      $ docker compose logs superexec -f
+
+#. Switch back to the second terminal and change the directory to the new quickstart
+   example you like to run:
+
+   .. code-block:: bash
+      :caption: Terminal 2
+
+      $ cd ../quickstart-pytorch
+
+#. Append the following lines to the end of the ``pyproject.toml`` file and save it:
+
+   .. code-block:: toml
+      :caption: pyproject.toml
+
+      [tool.flwr.federations.docker-compose]
+      address  =  "127.0.0.1:9093"
+      insecure  = true
+
+#. Run the example:
+
+   .. code-block:: bash
+      :caption: Terminal 2
+
+      $ flwr run . docker-compose
+
+Clean Up
+--------
+
+Remove all services and volumes:
+
+.. code-block:: bash
+
+   $ docker compose down -v
+
+Limitation
+----------
+
+.. list-table::
+   :header-rows: 1
+
+   * - Quickstart Example
+     - Limitation/Requirements
+   * - quickstart-fastai
+     - None
+   * - examples/quickstart-huggingface
+     - For CPU-only environments, it requires at least 32GB of memory.
+   * - quickstart-jax
+     - The example has not yet been updated to work with the latest ``flwr`` version.
+   * - quickstart-mlcube
+     - The example has not yet been updated to work with the latest ``flwr`` version.
+   * - quickstart-mlx
```

Comment:
We should remove the mentioning of Docker here: https://github.com/adap/flower/blob/541064b79e4de3d95be9ec270ff66da201feb3b6/examples/quickstart-mlx/README.md?plain=1#L70

Reply:
https://github.com/adap/flower/pull/4202